### PR TITLE
Merge master into v.next & fix up proj files

### PR DESCRIPTION
**Pull request by nCastle1**

### CODE REVIEW

**Comment by nCastle1** [comment]
@dkemlage I'm adding you as a reviewer because you're helping me test this. Thanks :)

**Comment by nCastle1** [comment]
For additional transparency, here's a list of less-straightforward things that also were changed while this merge was done:

- Includes in csproj/vbproj/projitems files for things like metadata, screenshots, resources were made consistent. E.g. all of the screenshots were marked as `None` for Android and iOS because those sample viewers don't use them
- Items were placed in `ItemGroup`s by type. This makes the msbuild (project) files easier to understand and manipulate programmatically. It also makes manual verification and merge-conflict resolution easier - there is now a 'right' answer as to where new files go
- Some changes that Visual Studio for Mac really wanted to make have been made. This will reduce the amount of cherry-picking of lines to commit when modifying the solutions on Mac. 
- A bug in the sample viewer projects for UWP led to Visual Studio occasionally throwing spurious duplicate resource errors. This has been resolved in the csproj/vbproj re-org. 
- Fewer items are copied around or included in executables unnecessarily. For example, the UWP has no dependency on images being copied to the output directory, so that doesn't happen anymore. 

Many of these changes were made to facilitate the work of resolving the merge conflicts now, but will also make life easier going forward.

**Comment by nCastle1** [comment]
@dkemlage Thanks for testing!

@ThadT Can you please review this for the changes that were made to the csproj/vbproj/projitems files? There is a summary of the changes/reasons for changes above.

File: src/Forms/Shared/Forms.projitems [hunk 1]
```diff
@@ -28,694 +34,677 @@
...
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Samples\Map\ManageBookmarks\metadata.json" />
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Samples\Map\OpenMapURL\metadata.json" />
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Samples\Map\OpenMobileMap\metadata.json" />
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Samples\Map\SearchPortalMaps\metadata.json" />
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Samples\Map\SetInitialMapArea\metadata.json" />
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Samples\Map\SetInitialMapLocation\metadata.json" />
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Samples\Map\SetMapSpatialReference\metadata.json" />
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Samples\Map\SetMinMaxScale\metadata.json" />
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Samples\MapView\ChangeViewpoint\metadata.json">
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
```
Comment (ThadT) [inline src/Forms/Shared/Forms.projitems]: Remove `CopyToOutputDirectory` element for this resource.

**Pull request merged**